src/test/java/HWTestForLoginTest.java

```diff
@@ -0,0 +1,66 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+    public class HWTestForLoginTest {
+    @Test
+    void test1 () throws InterruptedException {
+        //1. Login Success.
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://kwidos.com/");
+        Thread.sleep(2000);
+        driver.findElement(By.xpath("//*[@class='glyphicon glyphicon-log-in']/../p")).click();
+        Thread.sleep(2000);
+        driver.findElement(By.id("email")).sendKeys("alekseimaslo@gmail.com");
+        driver.findElement(By.id("password")).sendKeys("Kwidos123!");
+        driver.findElement(By.xpath("//*[@type='submit']")).click();
+
```

Comment:
Where is your assert? What does check this test? Please, add assert where you check that login is success